docs/docs/articles/test-executions.md

```diff
@@ -1,10 +1,10 @@
-# Test and Test Suite Execution CRDs
+# Test, Test Suite and Test Workflow Execution CRDs
```

Comment:
Worth to migrate it to [**testkube-docs**](https://github.com/kubeshop/testkube-docs)

Reply:
will copy